src/device/pf_pdport.c

```diff
@@ -81,6 +81,36 @@ static int pf_pdport_enadis(pnet_t *net, int loc_port_num, pf_link_state_link_t
    return 0;
 }
 
+/* XXX: FIXME refactor to a proper pnal_eth_pdport_dcp_boundary() function */
+/**
+ * Set or clear DCP boundary on port based on PDPortAdjsut:AdjustDCPBoundary
+ *
+ * @param   loc_port_num      In:   Local port number.
+ *                                  Valid range: 1 .. num_physical_ports
+ * @param   type              In:   Last octet in 01:0e:cf:00:00:0x (Ident/Hello)
+ * @param   block             In:   1: block egress of Ident(00) or Hello(01)
+ * @return  POSIX OK(0) or non-zero error
+ */
+static int pf_pdport_dcp_boundary(pnet_t *net, int loc_port_num, int type, int block)
+{
+   pf_port_t *p_port_data = pf_port_get_state (net, loc_port_num);
+   char *port = p_port_data->netif.name;
+   char group[20];
+   char cmd[256];
+
+   snprintf (group, sizeof(group), "01:0e:cf:00:00:%02x", type);
```

Comment:
Maybe good to have a "define" for this i.e. MULTICAST_GROUP_PATTERN, or to add a comment line above

Reply:
Ah good point, I'll add a reference to the spec.!